astrodendro/io/util.py

```diff
@@ -99,23 +96,18 @@ def _construct_tree(repr):
                 sub_structures = _construct_tree(sub_structures_repr)
                 for i in sub_structures:
                     d._structures_dict[i.idx] = i
-                b = Structure(structure_indices, f, children=sub_structures, idx=idx, dendrogram=d)
+                branch = Structure(structure_indices, f, children=sub_structures, idx=idx, dendrogram=d)
                 # Correct merge levels - complicated because of the
                 # order in which we are building the tree.
                 # What we do is look at the heights of this branch's
                 # 1st child as stored in the newick representation, and then
                 # work backwards to compute the merge level of this branch
-                first_child_repr = six.next(six.itervalues(sub_structures_repr))
```

Comment:
This removal looks like a substantive change; I'm not sure immediately that it's correct.

Reply:
The variable was unused, but I will double check whether this might have had any desirable side effects

Reply:
I can't see what purpose this code served (it was identified as unused during a style check)